packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -510,6 +507,21 @@ function makeAugmentedSchema(
         });
     });
 
+    function ensureNonEmptyInput(inputName: string | InputTypeComposer<any>) {
```

Comment:
`inputOrName` may be a slightly more accurate name for the param

 ```suggestion
     function ensureNonEmptyInput(inputOrName: string | InputTypeComposer<any>) {
```